doc/whatsnew/fragments/5701.other

```diff
@@ -0,0 +1,8 @@
+You can now set the ``files`` option in configuration files and on the command line.
+Passing files without the ``--files`` flag is still supported. This allows to set
+``files`` to ``files = my_source_directory`` and invoking ``pylint`` with only
+the ``pylint`` command similar to how other CLI tools allow to do so.
+The help message can always be invoked with ``pylint -h`` as well as with
+``pylint`` if the ``files`` option is not set.
```

Comment:
```suggestion
The help message can always be invoked with ``pylint -h`` or ``pylint --help``.
```
Probably better to not advertise this if we remove it in 3.0.

Reply:
Yeah, I'll do a full clean up of the PR to reflect the discussion and its result!